trulens_eval/trulens_eval/feedback/groundedness.py

```diff
@@ -97,9 +90,9 @@ def groundedness_measure(self, source: str, statement: str) -> float:
         )
 
         groundedness_scores = {}
-        if isinstance(self.groundedness_provider, (AzureOpenAI, OpenAI)):
+        if isinstance(self.groundedness_provider, (AzureOpenAI, OpenAI, LiteLLM, Bedrock)):
```

Comment:
Are we missing Hugging Face here? Or do we want LiteLLM to handle it?

Reply:
LiteLLM will handle LLMs from huggingface. If huggingface is set directly as the provider, we'll use the NLI model from hf.

Reply:
Oops I was only looking at the file changes. LGTM.